sample-compose/buildSrc/src/main/java/Versions.kt

```diff
@@ -1,33 +1,36 @@
 object Versions {
-    const val BUILD_GRADLE_VERSION = "4.2.1"
+    const val BUILD_GRADLE_VERSION = "7.3.0"
 
-    const val ANDROID_COMPILE_SDK_VERSION = 30
-    const val ANDROID_MIN_SDK_VERSION = 23
-    const val ANDROID_TARGET_SDK_VERSION = 30
+    const val ANDROID_COMPILE_SDK_VERSION = 33
+    const val ANDROID_MIN_SDK_VERSION = 24
+    const val ANDROID_TARGET_SDK_VERSION = 33
 
     const val ANDROID_VERSION_CODE = 1
     const val ANDROID_VERSION_NAME = "3.14.0"
 
     // Dependencies (Alphabet sorted)
     const val ANDROID_COMMON_KTX_VERSION = "0.1.1"
     const val ANDROID_CRYPTO_VERSION = "1.0.0"
-    const val ANDROIDX_ACTIVITY_KTX_VERSION = "1.2.1"
-    const val ANDROIDX_CORE_KTX_VERSION = "1.3.0"
-    const val ANDROIDX_FRAGMENT_VERSION = "1.3.3"
-    const val ANDROIDX_LIFECYCLE_VERSION = "2.4.0-alpha02"
+    const val ANDROIDX_ACTIVITY_KTX_VERSION = "1.6.0"
+    const val ANDROIDX_CORE_KTX_VERSION = "1.9.0"
+    const val ANDROIDX_TEST_CORE_VERSION = "1.4.0"
```

Comment:
Missing updates: 

- https://github.com/nimblehq/android-templates/pull/323/files#diff-8d83febc7bad9355f66dcdf21f1a25e2b5ee979000e8b11b4446d880542f969bR164

- https://github.com/nimblehq/android-templates/pull/323/files#diff-4c5e3948d43d0c3d46be7c596911cc18084a0281ccc7edccc19a1dfe6c49f0bdR32

Reply:
@Tuubz Did you mean adding `android:exported="true"`?

Reply:
This one 🙏 

![Screen Shot 2022-11-16 at 15 25 42](https://user-images.githubusercontent.com/18277915/202127616-0b16a3b7-80ca-4930-bc03-84fac5d54493.png)

Reply:
@Tuubz Updated 🙏